lib/presentation/screens/home/navigation/home_screen_router.dart

```diff
@@ -27,8 +29,11 @@ class HomeScreenRouter
 
     if (state is HomeChat) {
       pageStack.add(FadeTransitionPage.fromRoute(
-        route: AppRoutes.homeChatList,
-        child: const ChatScreen(),
+        route: AppRoutes.homeChat,
+        child: BlocProvider<ChatScreenCubit>(
```

Comment:
This Blocprovider shouldn't be there. It should be in `main_screen.dart` it's moved there but for you to know for this blueprint